matlab/vts_wrapper/vts_mc_demo.m

```diff
@@ -8,7 +8,7 @@
 startup();
 
 % ======================================================================= %
-%% Example 1: run a simple Monte Carlo simulation with 1000 photons
+%% Example 01: run a simple Monte Carlo simulation with 1000 photons
```

Comment:
Can we either remove the colon here or add it in the Examples below?

Reply:
the MC demos all had it, but I've updated the other scripts to match as well